docs/src/Examples/benders.md

```diff
@@ -0,0 +1,404 @@
+# Benders Quantile Regression
+
+We will apply Norm-1 regression to the [Linear Regression](https://en.wikipedia.org/wiki/Linear_regression) problem.
+Linear regression is a statistical tool to obtain the relation between one **dependent variable** and other **explanatory variables**.
+In other words, given a set of $n$ explanatory variables $X = \{ X_1, \dots, X_n \}$
+we would like to obtain the best possible estimate for $Y$.
+In order to accomplish such a task we make the hypothesis that $Y$
+is approximately linear function of $X$:
+
+$$Y = \sum_{j =1}^n \beta_j X_j + \varepsilon$$
+
+where $\varepsilon$ is some random error.
+
+The estimation of the $\beta$ values relies on observations of the variables:
+$\{y^i, x_1^i, \dots, x_n^i\}_i$.
+
+In this example we will solve a problem where the explanatory variables are sinusoids of differents frequencies.
+First, we define the number of explanatory variables and observations
+
+```julia
+using ParametricOptInterface,MathOptInterface,JuMP,GLPK
+using TimerOutputs,LinearAlgebra,Random
+
+const POI = ParametricOptInterface
+const MOI = MathOptInterface
+const OPTIMIZER = GLPK.Optimizer;
```

Comment:
Could you change for HiHGS? It is a minor thing but I think people are investing more on it nowadays